google-cloud-bigquery/acceptance/bigquery_helper.rb

```diff
@@ -123,6 +123,14 @@ def random_file_destination_name
       "kitten-test-data-#{SecureRandom.hex}.json"
     end
 
+    def assert_data data
+      assert_equal Google::Cloud::Bigquery::Data, data.class
+      refute_nil data.kind
+      refute_nil data.etag
+      refute_nil data.total
+      refute_nil data.count
```

Comment:
Why the change now to basic assertions?

Reply:
Because it is simpler and I didn't want to fight the Spec DSL to get these tests passing again.